cadc-dali/src/main/java/ca/nrc/cadc/dali/tables/votable/VOTableDocument.java

```diff
@@ -77,13 +77,14 @@
  */
 public class VOTableDocument {
 
-    private List<VOTableResource> resources = new ArrayList<VOTableResource>();
+    private List<VOTableResource> resources = new ArrayList<>();
+    private List<VOTableInfo> infos = new ArrayList<>();
 
     public VOTableDocument() {
     }
 
     public List<VOTableResource> getResources() {
-        return resources;
+        return this.resources;
```

Comment:
the explicit `this.` is not really needed here. It's useful when assigning values to member variables (because it allows argument and assignment to use the same name in ctor and set methods) but not otherwise.... especially in a simple get method where there is no other possibility.

ditto below in getInfos()

Reply:
Updated